pyproject.toml

```diff
@@ -41,11 +41,12 @@ dependencies = [
     "datalad>=0.15.4,<0.20",
     "pandas>=1.4.0,<2.2",
     "nibabel>=3.2.0,<5.11",
-    "nilearn>=0.9.0,<=0.11.0",
+    "nilearn>=0.9.0,<=0.10.2",
     "sqlalchemy>=1.4.27,<=2.1.0",
     "ruamel.yaml>=0.17,<0.18",
     "importlib_metadata; python_version<'3.10'",
     "h5py>=3.8.0,<3.10",
+    "httpx[http2]==0.26.0",
```

Comment:
Is it that strict the dependency?

Reply:
It's always recommended to keep it like so when dealing with libraries which aren't 1.0 yet as the APIs can change very often.